src/kOS.Safe/Encapsulation/ISerializableValue.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace kOS.Safe.Encapsulation
+{
+    public interface ISerializableValue
```

Comment:
I may be wrong, but I don't think that our serialization hooks into .net's serialization methods.  In this particular case, the interface is being used by @tomekpiotrowski to tell his serializer to reduce the structure to the built in type.

I haven't spent enough time with the serialization stuff to know how structures opt in (or out) of having suffixes serialized, so I don't know if the shift to the attribute system is worth the cost.